libvips/colour/colour.c

```diff
@@ -245,40 +242,43 @@ vips_colour_build(VipsObject *object)
 
 	if (vips_image_generate(out,
 			vips_start_many, vips_colour_gen, vips_stop_many,
-			in, colour)) {
-		g_object_unref(out);
```

Comment:
Was the removal of this unref intentional? I also noticed a missing unref in the error condition around line 240, i.e. if the `vips__profile_set()` call fails.

Reply:
Oop! No, a copy-paste error from an earlier version of this PR, I'll fix it, thanks!